### PR TITLE
Enhancement/errorgraphic

### PR DESCRIPTION
**Pull request by ZacharyBys**

### CODE REVIEW

File: frontend/src/components/ErrorGraphic.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        return (
+            <div className="error_container">
+                <img className="error_image" src={`http://${process.env.REACT_APP_IP}/res/error.png`} alt="Error"/>
+                <h1 className="error_message">{this.props.error_message} </h1>
+            </div>
+        )
+    }   
+}
+
+export default Error;
```
Comment (justin-cotarla) [inline frontend/src/components/ErrorGraphic.js]: Newlines at the end of js files

File: frontend/src/styles/Error.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    width: 250px;
+}
+
+.error_message {
+    text-align: center;
+    margin-top: 10px;
+    font-size: 35px;
+    color:  #1DE28F;
+    font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
+}
```
Comment (justin-cotarla) [inline frontend/src/styles/Error.css]: Give the css file the same name as the component

File: frontend/src/styles/Error.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    margin-top: 200px;
+    width: 250px;
+}
+
+.error_message {
+    text-align: center;
+    margin-top: 10px;
+    font-size: 35px;
+    color:  #1DE28F;
+    font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen-Sans, Ubuntu, Cantarell, Helvetica Neue, sans-serif;
```
Comment (justin-cotarla) [inline frontend/src/styles/Error.css]: What's this mess?

**Review by justin-cotarla — CHANGES_REQUESTED** [review]

File: frontend/src/styles/Error.css (hunk 3 above)
Comment (ZacharyBys) [reply frontend/src/styles/Error.css]: using the same font cascade as Jay for the profile page, specifying backups?

**Review by justin-cotarla — APPROVED** [review]
Looks good! Will make errors great again.

**Pull request merged**